src/objectTranslations/Contact-ca.objectTranslation

```diff
@@ -4444,6 +4444,12 @@
         <name>Primary_Household__c</name>
         <relationshipLabel>Contactes (llar primària)</relationshipLabel>
     </fields>
+    <fields>
+        <help><!-- The language the contact prefers to use. --></help>
```

Comment:
I thought I saw somewhere in reading this that the "contact" in the help should be "Contact" with a capital "C" -- maybe I misinterpreted but wanted to call out.

Reply:
Yep, I added a comment for myself further down in this diff to change it. Will do today.